crates/chia-consensus/src/gen/make_aggsig_final_message.rs

```diff
@@ -120,10 +116,10 @@ mod tests {
             hex!("4444444444444444444444444444444444444444444444444444444444444444").into();
         let puzzle_hash: Vec<u8> =
             hex!("3333333333333333333333333333333333333333333333333333333333333333").into();
-        let msg = b"message";
+        let mut msg = b"message".to_vec();
 
         let mut expected_result = Vec::<u8>::new();
-        expected_result.extend(msg);
+        expected_result.extend(msg.clone());
```

Comment:
doesn't really matter. but this should be `extend_from_slice()`

Reply:
Done - if you want to resolve this